internal/api/schema.go

```diff
@@ -41,11 +42,13 @@ func (a *API) HTTPUpload(w http.ResponseWriter, req *http.Request, pathParams ma
 
 	format := req.Header.Get("X-Format")
 	compatibility := req.Header.Get("X-Compatibility")
+	sourceUrl := req.Header.Get("X-SourceURL")
+	commitSHA := req.Header.Get("X-CommitSHA")
```

Comment:
All the above strings has to be constants, can we move this separate constant file?

Reply:
done moved to a file constants.go